libraries/resource_providers/api1000/synergy/storage_pool_provider.rb

```diff
@@ -0,0 +1,20 @@
+# (c) Copyright 2020 Hewlett Packard Enterprise Development LP
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed
+# under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+# CONDITIONS OF ANY KIND, either express or implied. See the License for the
+# specific language governing permissions and limitations under the License.
+
+module OneviewCookbook
+  module API1000
+    module Synergy
+      # StoragePool API500 C7000 provider
```

Comment:
update this comemnt to API1000 Synergy provider

Reply:
corrected